nio/router/base.py

```diff
@@ -58,7 +58,7 @@ def _block_defines_input_id(self, block):
             return True
         else:
             raise InvalidProcessSignalsSignature(
-                "Block {} signature is invalid".format(block.name()))
+                "Block {} signature is invalid".format(block.id()))
```

Comment:
I'm actually thinking for log messages and errors and all of that the name would still be useful. Maybe we include both? @hansmosh what do you think?

Reply:
Same as former comment, my thinking is that it should include both.